### ISSUE TITLE
Config node name conflict

### ISSUE THREAD

**Issue by dpslwk** [uhppoted/node-red-contrib-uhppoted#11]
Any chance you could rename the generically named `config` node to `uhppoted-config`
It causes a conflict with this package that has a Config node

<img width="710" alt="Config conlfict" src="https://user-images.githubusercontent.com/234460/185761887-7e4161b7-7181-48fa-864a-c9ec4a66ba8d.png">

Unfortunately I suspect this to be a braking change so should be a major version number bump
I have no idea if there is a way to migrate them in node-red

https://discourse.nodered.org/t/renaming-the-configuration-node-type-config-in-a-custom-node-best-practice/60041

**Comment by uhppoted** [comment]
Hmm - weird that it hasn't shown up before this! I'll have to look into it, but I suspect the short answer is going to be not easily. Unfortunately it would break a number of actively running systems that aren't mine so I'll have to figure out a migration path.

Am happy to create a separate branch with a renamed _config_ node if you don't mind installing a development version locally?

Also - any idea why it conflicts with _node-red-contrib-config_ ? That's typically just a local node created when somebody is experimenting.

**Comment by dpslwk** [comment]
`node-red-contrib-config` is another add-on package
I was developing on a flow on one system and found it a handy thing to install and use, unfortunately when it came time to migrate the flow to the production systems it conflicted with `node-red-contrib-uhppoted` which is already used in production for another flow

You can see here https://flows.nodered.org/node/node-red-contrib-config adds a node called `config` as apposed to a configuration node, most of which are named after what they are configuring 

<img width="511" alt="Screenshot 2022-08-21 at 08 58 00" src="https://user-images.githubusercontent.com/234460/185781539-e471f7c2-3018-47b7-830b-f85af7eb5c2f.png">

<img width="329" alt="Screenshot 2022-08-21 at 09 01 35" src="https://user-images.githubusercontent.com/234460/185781663-230ba35a-1230-41b5-bdab-122c2b68e74a.png">


Yeah I'd be be happy to test a branch, and do understand its a change that would require a bump to say v1.0.0 for everyone else 😉

**Comment by dpslwk** [comment]
also just found this 
https://flows.nodered.org/node/node-red-contrib-uhppoted/scorecard#

which shows the same conflict with a number of other packages

**Comment by uhppoted** [comment]
Great! Thanks for the info!

I should (hopefully) have a branch with the renamed _config_ node ready in a day or two and then will get started on figuring out the options around migrating existing users/flows.

**Comment by uhppoted** [comment]
Hi,

Have posted a provisional fix to branch [11-config-node-name-conflict](https://github.com/uhppoted/node-red-contrib-uhppoted/tree/11-config-node-name-conflict).

You'll have to install it locally and then update your flows to use the _uhppoted-config_ node - the easiest I found is just to edit the flow JSON file and change the configuration node type directly from:
```
    {
        ...
        "type": "config",
        ...
    },
```
to
```
    {  
        ...
        "type": "uhppoted-config",
       ...
    },
```

**Comment by uhppoted** [comment]
I think I originally assumed that NodeRED would namespace the nodes but it appears not and there are conflicts with other node names too.

Have created a discussion for the handful of other people that I'm aware of who are actively using this project, if you want to add any thoughts to that: [RFC: prefixing all nodes with uhppoted- to avoid naming conflicts](https://github.com/uhppoted/node-red-contrib-uhppoted/discussions/12)

**Comment by uhppoted** [comment]
Published v1.0.0 with the renamed nodes to [NodeRED](https://flows.nodered.org/node/node-red-contrib-uhppoted).

The [scorecard](https://flows.nodered.org/node/node-red-contrib-uhppoted/scorecard) isn't reporting any naming conflicts.

**Comment by uhppoted** [comment]
And (finally :-)) closing this ... feel free to reopen it if you need to.
